connectors/cli.py

```diff
@@ -89,14 +89,20 @@ async def _start_service(config, loop):
     services = [SyncService(config), JobCleanUpService(config)]
     for sig in (signal.SIGINT, signal.SIGTERM):
 
-        async def _shutdown(sig_name):
+        def _shutdown(sig_name):
             logger.info(f"Caught {sig_name}. Graceful shutdown.")
             for service in services:
-                logger.info(f"Shutdown {service.__class__.__name__}...")
-                await service.stop()
+                try:
```

Comment:
fly-by, you can use `service.name` now

Reply:
Hmmm, I tried but it did not work:

```
  File "/Users/artemshelkovnikov/git_tree/connectors-py/connectors/cli.py", line 109, in _shutdown
    f"Failed to correctly shutdown {service.name} due to an error: {e}"
AttributeError: 'SyncService' object has no attribute 'name'
```

I don't think our Service classes have a `name` property/attribute.